OpenRA.Mods.Common/Traits/Player/ConquestVictoryConditions.cs

```diff
@@ -26,6 +26,24 @@ public class ConquestVictoryConditionsInfo : ITraitInfo, Requires<MissionObjecti
 		[Desc("Disable the win/loss messages and audio notifications?")]
 		public readonly bool SuppressNotifications = false;
 
+		[Desc("The defeat announcer's name.")]
+		public readonly string DefeatAnnouncer = "Battlefield Control";
+
+		[Desc("The victory announcer's name.")]
+		public readonly string VictoryAnnouncer = "Battlefield Control";
+
+		[Desc("The victory announcer's color.")]
+		public readonly Color DefeatColor = Color.White;
+
+		[Desc("The defeat announcer's color.")]
+		public readonly Color VictoryColor = Color.White;
+
+		[Desc("The message which comes after a player won.'@' is the players name.")]
+		public readonly string VictoryMessage = "@ is victorious.";
+
+		[Desc("The message which comes after a player is defeated.'@' is the players name.")]
+		public readonly string DefeatMessage = "@ is defeated.";
```

Comment:
> We don't have a system in place to reference translatable strings from code, but I can guarantee that the eventual solution won't be to move the strings onto a trait with the [Translate] attribute (because that doesn't make sense for most of the cases). Extracting the strings like that here just makes the job harder for whoever decides to build our proper string database.

These extracted strings also implement a new and non-standard formatting placeholder.  When somebody comes along who does want to implement a proper translation database then they are going to have to remove these changes and then deal with breaking compatibility for anyone who has overriden these in their own mods.